misc/token.go

```diff
@@ -163,6 +164,57 @@ func GenerateAstarteJWTFromPEMKey(privateKeyPEM []byte, servicesAndClaims map[As
 	return token.String(), nil
 }
 
+// GetJWTAstarteClaims returns the set of Astarte claims for an Astarte Token.
+func GetJWTAstarteClaims(rawToken string) (AstarteClaims, error) {
```

Comment:
Why is this be public? I'd keep the public API as minimal as it's needed (i.e. `IsJWTAstarteClaimValidForService`), keeping this (and the `astarteClaims` struct) private. Every public function is a commitment to not break the API, so let's just expose what is needed right now.

Reply:
This is public so that we can use it in an astartectl command to show a token's claims